crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -251,6 +252,7 @@ impl ShaderCache {
         pipeline: CachedPipelineId,
         id: AssetId<Shader>,
         shader_defs: &[ShaderDefVal],
+        unchecked: bool,
```

Comment:
This functions should be unsafe as well since we do not run validation checks here.

Reply:
Following this logic, should I mark `process_compute_pipeline()`, `process_queue()`, `process_pipeline_queue_system()` etc. unsafe as well? They don't validate either 🤔